src/lib/button-outlined.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+	// export let label = '';
```

Comment:
@guacamole do we need this one? if not let's remove it

Reply:
you got it!